.circleci/config.yml

```diff
@@ -306,6 +315,7 @@ workflows:
       - unit_tests
       - cdlang_tests
       - coverage
+      - cpp-format-check
```

Comment:
We might want to set another environment variable to the job to determine the base branch. I think `pipeline.git.base_revision` or `pipeline.pull_request.target_branch` should do. Then this new variable can be used as the base ref in the diff command.

https://circleci-public.github.io/pipeline-preview-docs/pipeline_variables_overview.html
https://circleci-public.github.io/pipeline-preview-docs/pipeline_variables_reference.html

Reply:
If we don't want to add an environment variable, we could also pass the base ref as an arg to `check-cpp-format.sh`

Reply:
`pipeline.git.base_revision` is the immediate previous commit.

`pipeline.pull_request.target_branch` is not yet available.

Reply:
Could use `CIRCLE_PULL_REQUEST` and `GITHUB_TOKEN`

https://api.github.com/repos/stratum/stratum/pulls/396
`data.base.ref` contains the branch